packages/client/hmi-client/src/components/code/tera-code-dynamic.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+	<h3>{{ filename ?? '' }}</h3>
+	<Card
+		v-for="(block, index) in codefile?.dynamics?.block"
+		class="code-dynamic-card-container"
+		:key="index"
+	>
+		<template #title>
+			Code Block
+			<div v-if="!editable">
+				<!-- <Button text icon="pi pi-pencil" @click="editable = true" /> -->
```

Comment:
Why this is commented?

Reply:
I will be adding this in a follow-up PR and forgot to remove it.  Editing is going to be a little more difficult than i initially thought

Reply:
Ill hopefully have editing ready for monday...